app/lib/cognito.server.ts

```diff
@@ -162,6 +233,13 @@ export async function listGroupsForUser(Username: string) {
   return groups
 }
 
+export async function getUserGroupStrings(Username: string) {
+  const Groups = await listGroupsForUser(Username)
+  return Groups?.map(({ GroupName }) => GroupName).filter(Boolean) as
+    | string[]
+    | undefined
+}
```

Comment:
This function is only used in endorsements.server.ts. Would you please move it there?